packages/ketcher-macromolecules/src/components/modal/monomerConnection/MonomerConnections.tsx

```diff
@@ -200,6 +201,26 @@ function AttachmentPointSelectionPanel({
     const leavingGroup = MonomerCaps[attachmentPoint];
     return leavingGroup === 'O' ? 'OH' : (leavingGroup as LeavingGroup);
   };
+  let bond;
+  if (selectedAttachmentPoint) {
+    bond = monomer.getPotentialBond(selectedAttachmentPoint);
+  }
+
+  const handleSelectAttachmentPoint = (attachmentPoint: string) => {
+    const newBonds = { ...monomer.attachmentPointsToBonds };
+
+    if (selectedAttachmentPoint) {
+      newBonds[selectedAttachmentPoint] = null;
+    }
+
+    newBonds[attachmentPoint] = bond;
+    setBonds(newBonds);
+
+    onSelectAttachmentPoint(attachmentPoint);
+
+    const newConnectedAttachmentPoints = getConnectedAttachmentPoints(newBonds);
+    setConnectedAttachmentPoints(newConnectedAttachmentPoints);
+  };
```

Comment:
Could you please check why do the `R3` attachment point get disabled when double click the `R2` attachment point? And is this the expected behavior?
<img width="211" alt="image" src="https://github.com/epam/ketcher/assets/35161544/65c147ff-08fa-47c6-8072-42b1145f17f1">